editor/components/block-list/style.scss

```diff
@@ -376,7 +426,7 @@
 
 .editor-block-list__insertion-point-indicator {
 	position: absolute;
-	top: 16px;
+	top: $block-padding - 1px;	// Half the empty space between two blocks, minus the 2px height
```

Comment:
Nit: Tab before comment should be a space.

Reply:
👍 👍